docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md

```diff
@@ -11,6 +11,6 @@ Here is a list of Toolkit engine supported operating systems.
 
 ----------
 
-| Windows | macOS | CentOS |
+| Windows | macOS | Rocky Linux |
 |:-------:|:-----:|:------:|
-|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  7.4   |
\ No newline at end of file
+|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  8  |
```

Comment:
We still support CentOS 7. We just introduce support of Rocky 8 so we support them both at the moment.
We can have two columns, one for CentOS and one for Rocky if we want. Or only one "Linux" column and list Centos 7.4, Rocky 8.5 on the "version" line

Reply:
Thanks @julien-lang ! I'll update...